### ISSUE TITLE
Change "fierce leprechaun green" on Feedback/ Review portals to be a checkbox in a new column instead

### ISSUE THREAD

**Issue by rob0rt** [TexasLAN/Omega#45]

**Comment by rob0rt** [comment]
Changes will be here:
https://github.com/TexasLAN/Omega/blob/master/routes/review/ReviewListController.php
and here:
https://github.com/TexasLAN/Omega/blob/master/routes/feedback/FeedbackListController.php

Something like `<span class="glyphicon glyphicon-ok" aria-hidden="true"></span>` should be good, with maybe a bit of fierce leprechaun green coloring thrown on there.
